papaparse.js

```diff
@@ -530,7 +535,11 @@ License: MIT
 			if (IS_PAPA_WORKER)
 			{
 				global.postMessage({
-					results: results,
+					results: Object.assign(results, {
```

Comment:
@JohnLeSked Object.assign will mutate the first argument
so this whole block is similar to
```
result.meta.currentByteCursor = this._currentByteCursor
```

Reply:
Yeah, i put the wrong target, source order